QA.md

```diff
@@ -0,0 +1,59 @@
+# Hiring Quality Assurance Specialist for KodaDot NFT Gallery
+
+## About
+
+KodaDot, an NFT gallery that aims to be builder-owned and the ultimate public good, is always on the lookout for the best talent to join our growing team. We have a variety of opportunities available, including full-time, part-time, internships, freelance, and temporary gigs.
+
+## Responsibilities
+
+- Ensure the quality and reliability of the KodaDot NFT gallery through thorough testing
+- Work closely with the development team to identify, report, and resolve issues in a timely manner
+- Collaborate with the community, both in discussion and implementation of new features
+- Participate in code reviews, maintaining high code quality standards
+- Use various testing methodologies and tools to validate the functionality and performance of the product
+- Offload 30% of dev work by checking if specific PRs work as intended, reporting and raising bugs that break UX, identifying missing functionality that was accidentally removed
+- Write and update E2E and unit tests (using Vite)
+- Communicate with the team in issues to help offload major context switching from devs
+- Tech-savvy QA is preferred
+- Give "works-for-me" labels on PRs if things work as intended
+
+## Requirements
+
+- Strong proficiency in Node.js and adherence to its supported versions
+- Experience with pnpm as a package manager
+- Knowledge of Vue.js and familiarity with Vue Language Features (Volar) and Vue VSCode Snippets
+- Understanding of KodaDot's tech stack and infrastructure
+- Strong communication skills for effective collaboration with the community and team
+- Familiarity with KodaDot's coding conventions and recommendations
+
+## Opportunities at KodaDot
+
+- Full-time Quality Assurance Specialist
+- Part-time Quality Assurance Specialist
+- Quality Assurance Internship
+
+## Qualification for full-time, part-time & paid internships
+
+To be considered for these positions, candidates should have:
+
+- A strong track record on their GitHub profile, showcasing open-source contributions
+- Excellent cooperation and coordination skills within a team
+- Prior experience in quality assurance, testing methodologies, and tools
+- A fast learning curve and the ability to adapt to change requests quickly
+- A history of collaborating with development teams to improve product quality
+
+## How to apply?
+
+KodaDot offers paid interviews, valuing your time and contributions. You will be paid for completed pull requests (10-20) that are successfully merged. To start, focus on tackling issues with p1 or p2 labels, as they are often challenging and showcase your abilities. If you have any questions, feel free to [contact damsky](https://t.me/helloitsdamsky) on telegram. 
```

Comment:
Probably we can include somewhere between lines that helping testing PRs and labeling "works-for-me" probably I can add them to external-contributors for start to have labels iirc. Also bugs reporting could be do anyone. Yet we want to see Proof-of-Work. Don't want to take anyone who doesn't understand codebase.

From past I think we formulated this 
- https://docs.kodadot.xyz/contrib-helpers.html#catching-bugs
- Here I see QA paragraph https://docs.kodadot.xyz/contrib-helpers.html#rewards-1

Reply:
> Probably we can include somewhere between lines that helping testing PRs and labeling "works-for-me" probably

It’s in H2 - Give "works-for-me" labels on PRs if things work as intended

- Can also include https://docs.kodadot.xyz/contrib-helpers.html#quality-assurance into the .md file